src/js/api/build.js

```diff
@@ -39,6 +39,50 @@ ripe.Ripe.prototype.getBuildsP = function() {
     });
 };
 
+/**
+ * Retrieves the build artifact information by brand name and version.
+ *
+ * @param {String} name The name of the brand of the build artifact.
+ * @param {Number} version The number of the version of the build artifact.
+ * @param {Object} options An object of options to configure the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getBuildArtifacts = function(name, options, callback) {

```

Comment:
Shouldn't this simply be `getBuild`? The method for getting a build's artifacts already exists.